index.js

```diff
@@ -187,177 +387,206 @@ function mergeResults(contentArray) {
     return contentArray.reduce((merged, n) => merged.concat(n.content), []);
 }
 
-async function downloadInlineScan(version) {
+async function downloadGrype(version) {
+    var url;
     core.debug(`Installing ${version}`);
-    const downloadPath = await cache.downloadTool(`https://ci-tools.anchore.io/inline_scan-v${version}`);
+    
+    // Support using a branch instead of a version by (naively) checking if the version
```

Comment:
nit / non-blocking: recalling our previous conversations about testing from a branch, I agree with the usecase. As I'm reading this, a dev would need to gut the `install.sh` on their branch to either pull a binary from another location or to build from source in another way. Doesn't need to get solved necessarily now, but worth discussing later.

Reply:
I believe this comment doesn't reflect the current reality :( It isn't possible to use a branch, and `install.sh` will always pull the latest version if not specified. I didn't update the comment, I'll make sure I get this right before merging in. I agree though that it is super useful to test against unreleased versions

Reply:
This has been updated. Ensuring that the actual version is respected (if any). There is currently no support for dev/unreleased versions, so I added a comment.